<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOPortSelectDialog</class>
 <widget class="QDialog" name="IOPortSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="IO7radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO7</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO8radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>135</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO8</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO10radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>165</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO10</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO9radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO9</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO1radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO2radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>45</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO4radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>75</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO12radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>195</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO12</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO13radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO13</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO11radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO11</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO6radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>105</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO6</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO15radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>255</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO15</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO3radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>75</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO14radioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>225</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO14</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IONullradioButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>255</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 125, 125);</string>
   </property>
   <property name="text">
    <string> IO选择</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="IO5radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>105</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IO5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
